src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -263,7 +265,7 @@
      {:ref                     flat-list-ref
       :style                   [(style/channel-listing theme listing-height) channels-styles]
       :data                    flatten-channels
-      :content-container-style (when platform/ios? {:padding-bottom (safe-area/get-bottom)})
+      :content-container-style {:padding-bottom (safe-area/get-bottom)}
```

Comment:
This is also a nice example of how we can avoid leaking platform specifics in the styles 🙌